README.md

```diff
@@ -364,5 +364,22 @@ See the `config/stack/overrides` directory for details about where the
 overrides live. See the `stack.Makefile` for details about how
 `local-build` works.
 
+### Setup RBAC
+
+We can setup extra permissions to grant access to the resources which are not part of our stack. This can be done by specifing the permissions in the CustomResourceDefinition YAML of our custom resource.
```

Comment:
Would you be able to break the long lines earlier? (For example, see lines 363-365 above this.) It makes it a little easier to discuss the changes in GitHub, because comments are for a whole line, and the lines are smaller when there are more line breaks. Also, it'll keep the style in the document consistent : )

Reply:
Makes sense :+1: I will add more line breaks.

Reply:
Fixed